packages/react-email/src/utils/get-email-component.ts

```diff
@@ -56,6 +56,7 @@ export const getEmailComponent = async (
     ...global,
     console,
     Buffer,
+    URL,
     module: { exports: { default: undefined as unknown } },
     __filanem: emailPath,
```

Comment:
looks like typo to me, lol

Reply:
not a typo, not sure why this is missing in the global context, like a few other things, but this isn't a typo

Reply:
@gabrielmfern `__filanem: emailPath,` should be `__filename: emailPath,`

Reply:
hmm, yeah that's actually a typo, misread what you meant

thanks for pointing it out